docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-php.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Build a PHP application that uses YSQL
+headerTitle: Build a PHP application
+description: Build a small PHP application using the php-pgsql driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: PHP
+    identifier: cloud-php
+    weight: 900
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [PHP application](https://github.com/yugabyte/yugabyte-simple-php-app) that connects to a YugabyteDB cluster using the [php-pgsql](../../../../reference/drivers/ysql-client-drivers/#php-pgsql) driver and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in PHP.
+
+## Prerequisites
+
+- PHP runtime. The sample application was created using PHP 8.1 but should work with earlier and later versions. Homebrew users on macOS can install PHP using `brew install php`; the php-pgsql driver is installed automatically.
+- [php-pgsql driver](https://www.php.net/manual/en/pgsql.setup.php).
+  - Homebrew users of earlier PHP versions can install the driver using the `brew install phpXX-pdo-pgsql` command, where XX is the PHP version.
+  - Ubuntu users can install the driver using the `sudo apt-get install php-pgsql` command.
+  - CentOS users can install the driver using the `sudo yum install php-pgsql` command.
+
+### Yugabyte Cloud
+
+- You have a cluster deployed in Yugabyte Cloud. To get started, use the [Quick start](../../).
+- You downloaded the cluster CA certificate. Refer to [Download your cluster certificate](../../../cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate).
+- You have added your computer to the cluster IP allow list. Refer to [Assign IP Allow Lists](../../../cloud-secure-clusters/add-connections/).
+
+## Clone the application from GitHub
+
+Clone the sample application to your computer:
+
+```sh
+git clone https://github.com/yugabyte/yugabyte-simple-php-app && cd yugabyte-simple-php-app
+```
+
+## Provide connection parameters
+
+The application needs to establish a connection to the YugabyteDB cluster. To do this:
+
+1. Open the `sample-app.php` file.
+
+2. Set the following configuration-related constants:
+
+    - **HOST** - the host name of your YugabyteDB cluster. To obtain a Yugabyte Cloud cluster host name, sign in to Yugabyte Cloud, select your cluster on the **Clusters** page, and click **Settings**. The host is displayed under **Network Access**.
```

Comment:
I included them all more to explain each; perhaps not necessary. If we end up using these examples for Core as well, port would be the only one that never changes, but I feel it's useful to explain that 5433 is the default YSQL API port on all YBDBs; otherwise it's a question - why is this set to 5433?